### PR TITLE
general cleanups and better rescuing

### PR DESCRIPTION
**Pull request by colinsurprenant**
This is a general code cleanup and added better handling for the `rescue nil` statements.

### CODE REVIEW

File: lib/logstash/inputs/udp.rb [hunk 1]
```diff
@@ -39,35 +39,47 @@ class LogStash::Inputs::Udp < LogStash::Inputs::Base
...
-  public
   def register
     @udp = nil
-  end # def register
+  end
 
-  public
   def run(output_queue)
-  @output_queue = output_queue
+    @output_queue ||= output_queue
```
Comment (yaauie) [inline lib/logstash/inputs/udp.rb]: if `@output_queue` is already set, we will ignore the explicitly-given value?

I'm not sure I see the value of this change (and its mate on L45 where we initialize `@output_queue` with a `nil` value).

Comment (colinsurprenant) [reply lib/logstash/inputs/udp.rb]: yeah, I had a brain bubble for this one. For some reason I tough that the `run` method could be called multiple time and this change juste make sure we set `@output_queue` just once. in either ways both ways works but this change is useless. will revert, thanks for catching this.

**Comment by colinsurprenant** [comment]
@yaauie let me know how that looks now!

**Comment by colinsurprenant** [comment]
thanks @yaauie - will merge and push new version.

**Pull request merged**